package.json

```diff
@@ -19,70 +19,70 @@
+    "classnames": "~2.5.1",
     "cookie": "~0.6.0",
     "lodash": "~4.17.21",
-    "marked": "~11.1.0",
-    "react": "~18.2.0",
-    "react-bootstrap": "~2.9.1",
-    "react-dom": "~18.2.0",
-    "react-redux": "~9.0.4",
-    "react-router": "~6.21.0"
+    "marked": "~12.0.2",
+    "react": "~18.3.1",
+    "react-bootstrap": "~2.10.2",
+    "react-dom": "~18.3.1",
+    "react-redux": "~9.1.2",
+    "react-router": "~6.23.1"
   },
   "devDependencies": {
-    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.11",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.13",
     "@swc/cli": "^0.3.12",
     "@swc/core": "^1.5.5",
     "@swc/jest": "^0.2.36",
-    "@testing-library/jest-dom": "~6.1.5",
-    "@testing-library/react": "~14.1.2",
-    "@testing-library/user-event": "~14.5.1",
+    "@testing-library/jest-dom": "~6.4.5",
+    "@testing-library/react": "~15.0.7",
+    "@testing-library/user-event": "~14.5.2",
     "@types/cookie": "^0.6.0",
     "@types/jest": "^29.5.12",
     "@types/node": "^20.12.11",
-    "@types/react": "^18.3.1",
+    "@types/react": "^18.3.2",
     "@types/react-dom": "^18.3.0",
     "@typescript-eslint/eslint-plugin": "^6.21.0",
     "@typescript-eslint/parser": "^6.21.0",
-    "ajv": "~8.12.0",
+    "ajv": "~8.13.0",
     "circular-dependency-plugin": "~5.2.2",
-    "css-loader": "~6.8.1",
-    "eslint": "~8.56.0",
+    "css-loader": "~7.1.1",
+    "eslint": "~8.57.0",
     "eslint-config-vinta": "github:vintasoftware/eslint-config-vinta#9dd2803f4864a2e9942c636f78b367b4d5a56c67",
     "eslint-import-resolver-webpack": "~0.13.8",
     "eslint-plugin-import": "~2.29.1",
-    "eslint-plugin-jest": "~27.6.0",
+    "eslint-plugin-jest": "~27.9.0",
     "eslint-plugin-jsx-a11y": "~6.8.0",
-    "eslint-plugin-prettier": "~5.0.1",
+    "eslint-plugin-prettier": "~5.1.3",
     "eslint-plugin-promise": "~6.1.1",
-    "eslint-plugin-react": "~7.33.2",
-    "eslint-plugin-react-hooks": "~4.6.0",
+    "eslint-plugin-react": "~7.34.1",
+    "eslint-plugin-react-hooks": "~4.6.2",
     "eslint-plugin-sonarjs": "~0.23.0",
     "eslint-plugin-unicorn": "~49.0.0",
     "identity-obj-proxy": "~3.0.0",
     "jest": "~29.7.0",
     "jest-environment-jsdom": "~29.7.0",
     "jest-transform-stub": "^2.0.0",
-    "mini-css-extract-plugin": "~2.7.6",
-    "postcss": "~8.4.32",
-    "postcss-loader": "~7.3.3",
-    "prettier": "~3.1.1",
-    "react-refresh": "^0.14.0",
+    "mini-css-extract-plugin": "~2.9.0",
+    "postcss": "~8.4.38",
+    "postcss-loader": "~8.1.1",
+    "prettier": "~3.2.5",
+    "react-refresh": "^0.14.2",
     "redux-mock-store": "~1.5.4",
-    "sass": "~1.69.5",
-    "sass-loader": "~13.3.2",
-    "style-loader": "~3.3.3",
+    "sass": "~1.77.1",
+    "sass-loader": "~14.2.1",
+    "style-loader": "~4.0.0",
     "swc-loader": "^0.2.6",
     "typescript": "^5.4.5",
-    "webpack": "~5.89.0",
-    "webpack-bundle-tracker": "~3.0.0",
+    "webpack": "~5.91.0",
+    "webpack-bundle-tracker": "~3.1.0",
```

Comment:
Please upgrade webpack-loader as well. Their versions should always match.

Reply:
Done, thanks!